src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const weekDiscount = 50;
+  const threeDaysDiscount = 20;
+  const totalNoDiscount = price * days;
+
+  if (days >= 7) {
+    return totalNoDiscount - weekDiscount;
+  }
+
+  if (days < 7 && days >= 3) {
+    return totalNoDiscount - threeDaysDiscount;
+  } else {
+    return totalNoDiscount;
+  }
```

Comment:
the checklist, p.3

Reply:
Sorry, I dont know how I passed by checklist. Corrected everything